.storybook/helpers/button-class.helper.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export const BUTTON_TYPES = ['primary', 'success', 'warning', 'danger', 'neutral'];
+export const BUTTON_STYLES = ['outline', 'solid', 'flat'];
+
+export const buttonClassLoader = (buttonType, buttonStyle) => {
```

Comment:
```suggestion
export function getButtonClass({ buttonType, buttonStyle }: { buttonType: string, buttonStyle: string }) {
```

- I think `getButtonClass` is a better function name.
- I think it is more readable to use the `function` keyword instead of `const`.
- Add missing parameter type annotations.
- Use object destruction so that the parameters can't be passed in the incorrect order. Parameters of the same type are easy to transpose.

Reply:
Corrected in commit 99cc1ad

Reply:
Thanks!